src/runtime/divert.rs

```diff
@@ -17,43 +17,52 @@ pub enum TargetType {
     Path(Path),
 }
 
-#[derive(Debug, Deserialize, Default, PartialEq, Eq, Clone)]
+#[derive(Debug, Deserialize, PartialEq, Eq, Clone)]
 #[serde(from = "DivertData")]
 pub struct Divert {
-    pub target: Option<TargetType>,
+    pub target: TargetType,
     pub stack_push_type: PushPopType,
     pub pushes_to_stack: bool,
     pub external_args: Option<u32>,
-    pub is_external: bool,
     pub is_conditional: bool,
 }
 
 impl Divert {
-    pub fn new() -> Divert {
-        Divert::default()
+    pub fn new(target: TargetType) -> Divert {
+        Divert {
+            target,
+            stack_push_type: PushPopType::None,
+            pushes_to_stack: false,
+            external_args: None,
+            is_conditional: false,
+        }
+    }
+
+    pub fn new_divert(target: TargetType) -> Divert {
+        Divert::new(target)
     }
```

Comment:
I'm not really a fan of this function's name, I could see it getting confused with Divert::new rather easily, especially since it ends up just calling new anyways